test/integration/agent/egress_ip_assign_linux_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package agent
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"antrea.io/antrea/pkg/agent/controller/egress"
+	"antrea.io/antrea/pkg/apis/crd/v1alpha2"
+)
+
+const defaultEgressRunDir = "/var/run/antrea/egress"
```

Comment:
I think you could export that constant from `antrea.io/antrea/pkg/agent/controller/egress` to avoid having to redefine it here

Reply:
done